common/wuwebview/wuwebview.cpp

```diff
@@ -372,6 +372,7 @@ IPropertyTree *WuWebView::ensureManifest()
     if (!manifest)
     {
         StringBuffer xml;
+        // Is this threadsafe? Does it need to be? Should it use getEmbeddedManifestPTree ?
```

Comment:
It looks like WuWebView classes are created when needed, so not shared between threads.

Reply:
Possibly worth commenting to that effect